Makefile

```diff
@@ -1,41 +1,31 @@
-GIT_HASH:=$(shell git show --name-status | grep commit | awk '{print $$2}' | head -c6)
-VERSION:=$(shell grep "VERSION = " ./output/AdobeEdge.brs | sed -e 's/.*= //; s/,//g; s/"//g')
-
 # bsc: https://github.com/rokucommunity/brighterscript
 install-bsc:
 	(npm install brighterscript -g)
 
 clean:
-	(rm -rf ./output)
 	(rm -rf ./out)
-	
+
 archive:clean build-sdk
 	@echo "######################################################################"
 	@echo "##### Archiving AdobeEdge SDK"
 	@echo "######################################################################"
-	@echo git-hash=$(GIT_HASH)
-	@echo version=$(VERSION)
 
 	mkdir -p ./out
 
-	touch ./output/info.txt
-	@echo git-hash=$(GIT_HASH) >> ./output/info.txt
-	@echo version=$(VERSION) >> ./output/info.txt
```

Comment:
Any reason we need to remove this file?

Reply:
Yes, let's add it back. To find the version information, customers could refer to this file rather than searching for it in the brs file.